mobile/src/common/rounded-avatar.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import { View, Image } from 'react-native';
+
+export interface Props {
+    photoBase64: string;
```

Comment:
where does mime type go? Do you suggest to merge it in base64 outside of the component?